setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.1.0-beta.17
+current_version = 0.2.0
```

Comment:
Note: I'm gonna try something with this release. Basically, we'll remove `bumpversion` from our dependencies and just change the version in `setup.py` during PRs like these, and then create a tagged release through GitHub instead. The creation of a new tag should also trigger PyPI and DockerHub releases